test/test_namedtensor.py

```diff
@@ -290,22 +287,6 @@ def check_tuple_return(op, inputs, expected_names):
             check_tuple_return(F.max_pool2d_with_indices, [named_tensor_2d, [2, 2]], named_tensor_2d.names)
             check_tuple_return(F.max_pool3d_with_indices, [named_tensor_3d, [2, 2, 2]], named_tensor_3d.names)
 
-    def test_no_save_support(self):
```

Comment:
Instead of deleting these tests, could we turn them into passing test cases? I know there's a test in `test_serialization`, but the logic behind it is not entirely clear and there is some value in having some short tests here even if they do duplicate the `test_serialization` tests